src/test/java/de/eldoria/semvertools/util/IntegersTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-License-Identifier: MIT
+ *
+ * Copyright (c) 2021 SemVerTools team and contributors
+ */
+
+package de.eldoria.semvertools.util;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class IntegersTest {
+
+  @Test
+  void testParseEmpty() {
+    assertEquals(Optional.empty(), Integers.parseNonNegativeInt(""));
```

Comment:
All these empty optional equal checks can be simplified
```suggestion
    assertTrue(Integers.parseNonNegativeInt("").isEmpty());
```